.github/workflows/cid.yml

```diff
@@ -1,6 +1,12 @@
 name: CI/CD
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
```

Comment:
Undo. Or why?

Reply:
We can let CI run on master, PRs, and manually triggering by `workflow_dispatch`, no need double checks in every PR.

Reply:
Nah, there's no need to run manually because builds should be idempotent. And I want it to run on every branch because I don't use PRs for my own changes.

Revert.

Reply:
2abd688491737450ed91a851264a4182d203b64d